backend/apps/migrations/management/commands/apply_migration_phase2.py

```
@@ -0,0 +1,11 @@
+class Command(BaseCommand):
```

Ruff (0.15.21)

[error] Undefined name `BaseCommand`

(F821)